source/Makefile

```diff
@@ -4,7 +4,12 @@ JSON_FILES=$(patsubst %.yml,%.json,$(YAML_FILES))
 all: $(JSON_FILES) HAS_JSYAML
 
 %.json: %.yml
-	js-yaml $< > $@
+	tmpfile=$$(mktemp); \
+	if js-yaml $< > "$$tmpfile"; then \
+	    mv "$$tmpfile" $@; \
+	else \
+	    rm "$$tmpfile"; \
```

Comment:
Please add here:
```
	    exit 1; \
```
... so that "make" has a non-zero exit code if js-yaml fails.